spec/lib/bank_account/savings_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'bank_account'
```

Comment:
This is nit-picky, but this file's name should follow the same naming convention as the file you're testing (i.e. `savings_account_spec.rb` for `savings_account.rb`)